crates/biome_js_parser/test_data/inline/err/import_invalid_args.js

```diff
@@ -1,3 +1,4 @@
 import()
 import(...["foo"])
 import("foo", { assert: { type: 'json' } }, "bar")
+import("foo", { with: { type: 'json' } }, "bar")
```

Comment:
This is now considered correct? We should move it to `ok/import_call.js`.

Reply:
No, that is wrong, it seems that `import_attributes` only have two params: https://github.com/tc39/proposal-import-attributes?tab=readme-ov-file#dynamic-import

Reply:
Maybe you can provide some reference documentation about this place?